server/seeders/20181107135821-roles.js

```diff
@@ -0,0 +1,18 @@
+export default {
+  // eslint-disable-next-line max-len
+  up: (queryInterface, Sequelize) => queryInterface.bulkInsert('Roles', [{
```

Comment:
Good job. I like the fact that you are using seed data rather than using mock data.